docs/deploy-development-rp.md

```diff
@@ -419,3 +419,51 @@ To run fake metrics socket:
 ```bash
 go run ./hack/monitor
 ```
+## Run the RP and create a Hive cluster
+
+**Steps to perform on Mac**
+
+1. Mount your local MacOS filesystem into the podman machine:
+`podman machine init --now --cpus=4 --memory=4096 -v $HOME:$HOME`
+
+2. Use the openvpn config file (which is now mounted inside the podman machine) to start the VPN connection:
+`podman machine ssh`
+`sudo rpm-ostree install openvpn`
+`sudo systemctl reboot`
+`podman machine ssh`
+`sudo openvpn --config /Users/<user_name>/go/src/github.com/Azure/ARO-RP/secrets/vpn-aks-westeurope.ovpn --daemon --writepid vpnpid`
+`ps aux | grep openvpn`
+
+### Instructions for Modifying Environment File
+
+**Update the env File**
+
+- Open the `env` file.
+-  Set `OPENSHIFT_VERSION ` to the desired OpenShift version, e.g., `4.12.25 `.
+-  Update INSTALLER_PULLSPEC with the appropriate name and tag, typically matching the OpenShift version, e.g., `release-4.12.`(for more detail see the `env.example`)
```

Comment:
We should mention the OCP_PULLSPEC env var also needs to be updated.  That's mentioned in env.example, but someone might miss that going back and forth from the readme to the env file.

Also, it would help to mention quay.io/openshift-release-dev/ocp-release and filtering the tags to look for the hash for a specific release version.  See my other comment.